### DISCUSSION TITLE
How to add a new control?

### DISCUSSION THREAD

**Discussion by Tyroo** [apache/superset#13404] [Q&A / Help]
Now I need to add a new control to the query tab of the control panel. My operation steps are as follows:

1.To "src/plugin/ControlPanel.ts” Add the following
![image](https://user-images.githubusercontent.com/62151691/109592638-8ab75880-7b4a-11eb-87e6-01ce635fda35.png)

2.To "superset-frontend/src/explore/controls.jsx” Add the following
![image](https://user-images.githubusercontent.com/62151691/109593074-4ed0c300-7b4b-11eb-907f-fce91af1d2a2.png)

3.Finally, the run throws the following error
![image](https://user-images.githubusercontent.com/62151691/109593436-ecc48d80-7b4b-11eb-991d-98afdb00ae09.png)

How can I solve this problem, thank you!

**Comment by mistercrunch** [comment]
Tagging @rusackas who can point to the right place

**Comment by webobite** [comment]
Hi @Tyroo / @rusackas  

Any fix available for this issue ?
